cityofaustin/atd-moped
# [Enhancement] Track if phase start and end dates are tentative or confirmed

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/14300

## Testing
**URL to test:** 


local! 

**Steps to test:**

`select * from moped_proj_phases where is_phase_end_confirmed is true or is_phase_start_confirmed is true`


From the timelines tab:
1. add a new phase. set the start date and/or the end date to a date in the past. Run the above query in your db client to see if your phase appears. 
2. edit an existing phase! see if it appears in the results of your query
3. repeat the steps with a date in the future. phase end and phase start confirmed is false

---
#### Ship list
